### PR TITLE
namespacing by class prefix

### LINKED ISSUE #90: namespace prefix?
https://luckyframework.org/guides/http-and-routing/routing-and-params#namespaces-are-handled-automatically
> You can namespace your actions by creating subfolders like src/actions/admin/projects/index.cr.
> class Admin::Projects::Index < BrowserAction

Is it really the folder, or rather the Admin:: prefix of the class name?

### PR DESCRIPTION
**Pull request by testbird**
closes: https://github.com/luckyframework/website/issues/90

### CODE REVIEW

File: src/actions/guides/http_and_routing/routing_and_params.cr [hunk 1]
```diff
@@ -179,7 +179,7 @@ class Guides::HttpAndRouting::RoutingAndParams < GuideAction
 
     ### Namespaces are handled automatically
 
-    You can namespace your actions by creating subfolders like `src/actions/admin/projects/index.cr`.
+    You can namespace your routes by prefixing the class name, e.g. with `Admin::`.
```
Comment (paulcsmith) [inline src/actions/guides/http_and_routing/routing_and_params.cr:182]: Would be good to put the filename on line 185 as a comment I think: `# in src/actions/admin/projects/index.cr`

**Review by paulcsmith — COMMENTED** [review]
Just one comment, then looks good to merge

**Comment by testbird** [comment]
BTW: Ok, but the file location would not need to change, right. Simply changing the class name is a dynamic feature that might get lost with moving the inference to the generator.

**Comment by paulcsmith** [comment]
Correct. The file path does not need to change it’s just there for convention

**Pull request merged**